source/nodejs/adaptivecards-site/service-worker.js

```diff
@@ -0,0 +1,109 @@
+// Used pwa-builder as reference: https://github.com/pwa-builder/pwabuilder-serviceworkers/blob/master/serviceWorker5/pwabuilder-sw.js
+importScripts(
+  'https://storage.googleapis.com/workbox-cdn/releases/6.4.1/workbox-sw.js'
```

Comment:
I think we'll want to pull this from a different CDN, or perhaps we should host it directly on our site?

Reply:
@paulcam206 I updated to use local workbox files per this documentation: https://developer.chrome.com/docs/workbox/modules/workbox-sw/#using-local-workbox-files-instead-of-cdn

Let me know what you think!